src/unix/linux_like/linux/musl/b64/mips64.rs

```diff
@@ -4,7 +4,7 @@ use crate::prelude::*;
 pub type wchar_t = i32;
 pub type __u64 = c_ulong;
 pub type __s64 = c_long;
-pub type nlink_t = u64;
+pub type nlink_t = u32;
```

Comment:
Mind using `c_uint` instead to match the source `unsigned`?

Reply:
Done :)